README.md

```diff
@@ -1,3 +1,7 @@
+> [!WARNING]  
+> This repository is no longer actively maintained. It is strongly suggested that you use the [officially-supported ruff action from Astral](https://github.com/astral-sh/ruff-action).
```

Comment:
This should either capitalize `Ruff` to match our official styling or say `ruff-action`.

Reply:
Done